style.css

```diff
@@ -0,0 +1,66 @@
+@import url('https://fonts.googleapis.com/css2?family=K2D&display=swap');
+
+.watch_allover {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 600px;
+    height: 700px;
+    background: #222222;
+    border: 10px solid #828282;
+    border-radius: 50px;
+}
+
+.watch_time {
+    display: flex;
+    width: 100%;
+    justify-content: end;
+}
+
+.watch_time .hour {
+    font-family: 'K2D';
```

Comment:
add styles with * selector
 
 font-family: 'K2D';

add to this selector * the style for the font. why use it everywhere...